### ISSUE TITLE
the border will appear misaligned when emoji was used in the title

### ISSUE THREAD

**Issue by ThaddeusJiang** [fuelen/owl#21]
![image](https://github.com/fuelen/owl/assets/17308201/3f3661ea-ebbf-4d0f-997c-bf2af0f3770e)

```elixir
iex(1)> "Owl" |> Owl.Box.new(title: "hello") |> Owl.IO.puts()
┌─hello────┐
│Owl       │
└──────────┘
:ok
iex(2)> "Owl" |> Owl.Box.new(title: "😂 hello") |> Owl.IO.puts()
┌─😂 hello────┐
│Owl         │
└────────────┘
:ok
```

**Comment by fuelen** [comment]
This is not only about borders :)
```
> "😂 Owl" |> Owl.Box.new(title: "😂 hello") |> Owl.IO.puts()
┌─😂 hello────┐
│😂 Owl       │
└────────────┘
```

**Comment by fuelen** [comment]
@ThaddeusJiang could you check in `main`?

**Comment by ThaddeusJiang** [comment]
@fuelen Yes, it works. Thank you every much.

**Comment by fuelen** [comment]
Published `v0.9.0`
